app/auth/views/feedback.py

```diff
@@ -5,20 +5,24 @@
 from ...utils import (
     get_json_from_request
 )
+from dmapiclient.audit import AuditTypes
 
 
-@login_required
 @auth.route('/feedback', methods=["POST"])
+@login_required
 def post_feedback():
     feedback_data = get_json_from_request()
 
+    if feedback_data['object_type'] == 'Brief':
```

Comment:
we use the audit table to save brief feedback?  Shouldn't that be its own table?

Reply:
even in the UK version, it is used as generic blob storage https://github.com/alphagov/digitalmarketplace-api/blob/master/scripts/snapshot_framework_stats.py

Reply:
yeah I think I remember have the same discussion re stats

I suppose there's pros/cons either way